### ISSUE TITLE
Job Scheduling - Error: undefined

### ISSUE THREAD

**Issue by SEyEAKey** [aws-samples/pcluster-manager#125]
Receiving "Error: undefined" when selecting "Job Scheduling" menu under a cluster. This seemed to be fixed by adding an additional policy to the head node 

- Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore

After adding that it still does not see the Slurm queues though after clicking the "Submit Job" button. 

Shows "Queue" and "[ANY]"

![image](https://user-images.githubusercontent.com/104110509/164344255-ba31efd8-5bb5-482d-b33e-b9105d0ea039.png)

**Comment by sean-smith** [comment]
Is this in `us-gov-west-1`?

**Comment by SEyEAKey** [comment]
us-east-2 (ohio)

**Comment by charlesg3** [comment]
@SEyEAKey  -- thanks for your interest... Good catch, the tab is updated so that it will not try to do any scheduling operations (e.g. submit jobs or poll the queue(s)) if you do not have SSM enabled.

As for the reason why it doesn't have the list of queues -- I'm a bit puzzled. Was this perhaps gathered before the cluster had been fully created?

**Comment by SEyEAKey** [comment]
I was working directly with @sean-smith and he solved the SSM issue.

I just noticed that on another cluster I created, it is showing the queues. Both were created a few hours ago.

There are a few differences between the two clusters.

- Different instance type for the head node
- The cluster where the queues are not showing the compute instances are in a private subnet (working cluster has the compute instances in the same subnet as the head)
- Compute instance types are different
